### PR TITLE
Lci

### PR DESCRIPTION
**Pull request by kyl0049**
Finished all LCI results algorithms.

### CODE REVIEW

File: methods/lcarun.ts [hunk 1]
```diff
@@ -2,28 +2,102 @@ import knex from 'knex';
...
+
+// NonrenewableUnitParams were used to change units for nonrenewable group. Units change is not needed for other groups
+interface NonrenewableUnitParams {
+  'brownCoal': number;
+  'hardCoal': number;
+  'crudeOil': number;
+  'mineGas': number;
+  'naturalGas': number;
+  'Uranium': number;
+  [key: string]: number;
```
Comment (laholstege) [inline methods/lcarun.ts]: What is this used for?

File: methods/lcarun.ts [hunk 2]
```diff
@@ -2,28 +2,102 @@ import knex from 'knex';
...
+// tslint:disable-next-line: variable-name
+  let COsum = 0;
+// tslint:disable-next-line: variable-name
+  let NOxsum = 0;
+// tslint:disable-next-line: variable-name
+  let NMVOCsum = 0;
+  let particulatesSum = 0;
+// tslint:disable-next-line: variable-name
+  let CO2eSum = 0;
+  for (let i = 0; i < inputRows.length; i++) {
```
Comment (laholstege) [inline methods/lcarun.ts]: I would break these into two different functions

File: methods/lcarun.ts [hunk 3]
```diff
@@ -2,28 +2,102 @@ import knex from 'knex';
...
+                                             'mineGas': 32.45, 'naturalGas': 32.12, 'Uranium': 560000};
 
 const lcarun = async (params: RunParams, db: knex) => {
-  const rows: Lci[] = await db
+  const inputRows: Lci[] = await db
     .table('lci_input')
-    .where({ lci_group: 'renewable' });
+    .where({ lci_group: 'renewable' }).orWhere({ lci_group: 'nonrenewable' }).orWhere({ lci_group: 'water' });
+
+  const outputRows: Lci[] = await db
```
Comment (laholstege) [inline methods/lcarun.ts]: the naming here is kind of confusing

File: uninstall-node.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```
Comment (laholstege) [inline uninstall-node.sh]: you don't need to push this file

File: methods/lcarun.ts (hunk 3 above)
Comment (kyl0049) [reply methods/lcarun.ts]: renamed in new branch KL/lci_claculation

File: methods/lcarun.ts (hunk 2 above)
Comment (kyl0049) [reply methods/lcarun.ts]: did that in the new branch, KL/lci_calculation.

File: methods/lcarun.ts (hunk 1 above)
Comment (kyl0049) [reply methods/lcarun.ts]: add comments for this part in the new branch, KL/lci_calculation

**Comment by kyl0049** [comment]
Please check all new changes in the new branch, KL/lci_calculation.

**Pull request closed without merging**